angularjs-portal-mock-portal/src/main/webapp/web/layoutDoc

```diff
@@ -169,9 +169,9 @@
          "faIcon":"fa-clock-o",
          "fname":"leave-statement",
          "target":null,
-         "widgetURL":null,
-         "widgetType":null,
-         "widgetTemplate":null,
+         "widgetURL":"/web/samples/leave-statement.json",
+         "widgetType": "generic",
+         "widgetTemplate": "<div ng-init='show=false'><div class='container-fluid ' ng-show='!isEmpty' > <div class='row'> <div class='col-xs-2'>&nbsp;</div> <div class='col-xs-8 center'> <button ng-click='show=!show' class='btn btn-flat'><span ng-if='!show'>Show</span><span ng-if='show'>Hide</span> balances</button></div> <div class='col-xs-2'>&nbsp;</div></div></div></div> <div><loading-gif data-object='content' data-empty='isEmpty'></loading-gif><div ng-if='isEmpty' style='padding: 10px; font-size: 14px;'><i class='fa fa-exclamation-triangle fa-3x pull-left' style='color: #b70101;'></i><span style='color: #898989;'>We had a problem fetching your balances.</span></div></div> <ul class='widget-list'> <li ng-repeat=\"item in filteredArray(content.report,'entitlement',['Sick','Vacation Ava','Sabbatical', 'Personal']) | orderBy:'entitlement' | limitTo:5\" ng-init='details=false' ng-mouseenter='details=true' ng-mouseleave='details=false'> <p class='bold'>{{item.entitlement}}<span class='right' ng-if='show'>{{item.balance}}</span><span class='right' style='font-style: italic; font-weight: 100;' ng-if='!show'>hidden</span> </p><p ng-if='details'>{{item.job.title}}</p> </li> <li ng-if='content.report.length > 5' style='font-style: italic; text-align: center;'>Showing 5 out of {{ content.report.length}} </li> </ul></div><a class='btn btn-default launch-app-button ng-scope' href='{{portlet.url}}'>See all balances</a>",
```

Comment:
Pretty view of template can be found [here](https://gist.github.com/timlevett/61951dc9f51e38ba0dfd)